### PR TITLE
make tabulation command configurable

### PR DESCRIPTION
**Pull request by rolandwalker**
The `column` command has a rather unfortunate line-length limit of 2048 on maxOS:

```bash
$ perl -E 'say q{_} x 2048' | column
column: line too long
$ sysctl user.line_max
user.line_max: 2048
```

which can be worked around by making the tabulation command configurable, after which a user with long history lines might choose to *eg*

```bash
HISTDB_TABULATE_CMD=(sed -e $'s/\x1f/\t/g')
```

or even (more hackishly)

```bash
HISTDB_TABULATE_CMD=(sed -e $'s/.*\x1f//')
```

### CODE REVIEW

**Comment by larkery** [comment]
That's a weird limitation - I am disappointed the fix is required but the implementation is fine.

**Comment by mafredri** [comment]
This limitation was a bit annoying to me, and the `sed` solution did make it hard to parse the output. So I implemented a custom tabulation command in zsh. I thought I'd share it here in case anybody feels the same way.

It's a bit long, so I created a gist for it.

https://gist.github.com/mafredri/076b22e321be4908ebf6170572b3545b
